tests/common.bash

```diff
@@ -45,3 +64,160 @@ waitForProcess() {
 	done
 	return 1
 }
+
+# Check if the $1 argument is the name of a 'known'
+# Kata runtime. Of course, the end user can choose any name they
+# want in reality, but this function knows the names of the default
+# and recommended Kata docker runtime install names.
+is_a_kata_runtime() {
+	if [ "$1" = "containerd-shim-kata-v2" ] || [ "$1" = "io.containerd.kata.v2" ]; then
+		echo "1"
+	else
+		echo "0"
+	fi
+}
+
+# Gets versions and paths of all the components
+# list in kata-env
+extract_kata_env() {
+	RUNTIME_CONFIG_PATH=$(kata-runtime kata-env --json | jq -r .Runtime.Config.Path)
+	RUNTIME_VERSION=$(kata-runtime kata-env --json | jq -r .Runtime.Version | grep Semver | cut -d'"' -f4)
+	RUNTIME_COMMIT=$(kata-runtime kata-env --json | jq -r .Runtime.Version | grep Commit | cut -d'"' -f4)
+	RUNTIME_PATH=$(kata-runtime kata-env --json | jq -r .Runtime.Path)
+
+	# Shimv2 path is being affected by https://github.com/kata-containers/kata-containers/issues/1151
+	SHIM_PATH=$(readlink $(command -v containerd-shim-kata-v2))
+	SHIM_VERSION=${RUNTIME_VERSION}
+
+	HYPERVISOR_PATH=$(kata-runtime kata-env --json | jq -r .Hypervisor.Path)
+	# TODO: there is no kata-runtime of rust version currently
+	if [ "${KATA_HYPERVISOR}" != "dragonball" ]; then
+		HYPERVISOR_VERSION=$(${HYPERVISOR_PATH} --version | head -n1)
+	fi
+	VIRTIOFSD_PATH=$(kata-runtime kata-env --json | jq -r .Hypervisor.VirtioFSDaemon)
+
+	INITRD_PATH=$(kata-runtime kata-env --json | jq -r .Initrd.Path)
+}
+
+# Checks that processes are not running
+check_processes() {
```

Comment:
Where is it going to be used?

Reply:
The following metrics make use of check_processes():
- disk/cassandra_kubernetes/cassandra.sh
- storage/fio-k8s/fio-test-ci.sh
- network/iperf3_kubernetes/k8s-network-metrics-iperf3.sh
- density/k8s-sysbench.sh